src/plugins/postal-savings-rs/preferences.xml

```diff
@@ -19,6 +19,16 @@
         negativeButtonText="Cancel"
         summary="||***********"
     />
+    <EditTextPreference
+        key="cardNumber"
+        obligatory="false"
+        inputType="text"
+        title="Card Number"
+        dialogTitle="Card Number"
+        positiveButtonText="OK"
+        negativeButtonText="Cancel"
+        summary="||{@s}"
```

Comment:
Haven't found any documentation for this field. What should be here?

Reply:
It's a legacy field. Should be preserved as it is :)